portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +36,10 @@ function FloatingNavbar() {
   else {
     navbar.classList.remove("sticky");
   }
+}
+
+function getBookRec() {
+  fetch('/literature').then(response => response.text()).then((quote) => {
```

Comment:
Ideally you would have some error handling logic here in case you don't get an expected response

Reply:
Oops, I just followed the example code snippet - I'll look into HTTP error handling!